src/controllers/mockController.js

```diff
@@ -0,0 +1,32 @@
+import { Notification, Helpers } from '../utils';
+
+const { notify } = Notification;
+const { successResponse, errorResponse } = Helpers;
+/**
+ * A collection of methods that mocks specific functionalities on the App.
+ * @class MockController
+ */
+class MockController {
```

Comment:
Why is this called mock controller? Is notification controller not more suitable?

Reply:
The reason for this is because we are yet to implement any notification feature, therefore, we currently don't have a notification controller. Since this route was created for the sole purpose of testing the notification methods I added (the route is only necessary for this reason) while waiting for an actual feature that builds on top of it, I thought It'd be okay to place it in a mock file instead. the team also agreed on moving existing mocks and adding new ones into these files. whats your advice? do I create a notification controller instead?

Reply:
Alright, we can leave it